tests/wicked/2nics/sut/t03_setup_second_card.pm

```diff
@@ -0,0 +1,42 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Set up a second card
+# Maintainer: Anton Smorodskyi <asmorodskyi@suse.com>
+#             Jose Lausuch <jalausuch@suse.com>
+#             Clemens Famulla-Conrad <cfamullaconrad@suse.de>
+
+use base 'wickedbase';
+use strict;
+use warnings;
+use testapi;
+use utils 'file_content_replace';
+use network_utils 'iface';
+
+
+sub run {
+    my ($self, $ctx) = @_;
+    my $config  = '/etc/sysconfig/network/ifcfg-' . $ctx->iface();
+    my $config2 = '/etc/sysconfig/network/ifcfg-' . $ctx->iface2();
+    record_info('Info', 'Set up a second card');
+    $self->get_from_data('wicked/static_address/ifcfg-eth0_second_card', $config);
+    $self->get_from_data('wicked/dynamic_address/ifcfg-eth0',            $config2);
```

Comment:
Is there a reason, why you are using the first interface with `second_card` config?

Reply:
static_address folder already contains `ifcfg-eth0` file but I can't reuse it because it has config for `10.0.2.11`. and don't want to go into having template here because this will mean that I would need to replace not only IP but also BROADCAST and NETWORK variables which force me to create code to manage 6 variables and modifications in wicked_basic . so as 'cheapest' approach I choose just add extra file containing exact  configuration as needed